.azure-pipelines/client.yml

```diff
@@ -99,4 +103,14 @@ jobs:
           AZ_CONFIG_CONNECTION: $(AzConfigConnectionString)
         inputs:
           command: test
-          projects: '$(ProjectFile)'
\ No newline at end of file
+          projects: '$(ProjectFile)'
+          arguments: --logger trx
+          publishTestResults: false
+
+      - task: PublishTestResults@2
+        condition: succeededOrFailed()
+        inputs:
+          testResultsFiles: '**/*.trx'
+          testRunTitle: '$(OSName) DotNet $(DotNetCoreVersion)'
```

Comment:
After looking at the test results I noticed that you specified the DotNetCoreVersion here but that isn't a variable that is currently changing and in general isn't that interesting in this context. However for now it is fine but we will probably want to adjust this if we later add other variables to the matrix strategy.